gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py

```diff
@@ -3995,3 +3994,12 @@ def impl(context):
     cmd =  "sudo mv -f /tmp/hosts_orig /etc/hosts; rm -f /tmp/clusterConfigFile-1; rm -f /tmp/hostfile--1"
     context.execute_steps(u'''Then the user runs command "%s"''' % cmd)
 
+@given('"LC_ALL" is different from English')
+def step_impl(context):
+	os.system('sudo locale-gen ru_RU.UTF-8 > /dev/null')
+	os.environ['LC_ALL'] = 'ru_RU.utf8'
+
+@then('gpstop should not print "Failed to kill processes for segment"')
+def impl(context):
+	check_string_not_present_stdout(context, 'Failed to kill processes for segment')
+	del os.environ['LC_ALL']
```

Comment:
what if `os.environ['LC_ALL'] is not None` before setting it to ru_RU.utf8?

Reply:
fixed